Error version 2.4.0 ECMAScript

Hello, in version 2.4.0 it gives me the following error:

`[ng] ./node_modules/object-sizeof/indexv2.js:5:13-28 - Error: Module not found: Error: Can't resolve 'util' in 'DIR\node_modules\object-sizeof'
[ng]
[ng] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[ng] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[ng]
[ng] If you want to include a polyfill, you need to:
[ng]    - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
[ng]    - install 'util'
[ng] If you don't want to include a polyfill, you can use an empty module like this:
[ng]    resolve.fallback: { "util": false }
[ng]
[ng]
[ng] × Failed to compile.`

In version 2.3.0 it seems to work fine.

greetings, thank you!

Hello, it seems that it is still happening in version 2.4.1 but in another line, I think that isNode() is not doing its job well. Same is better to know if you have "window" object for browsers and "global" for node? Although this could change in the future. Or does the same thing have to do with it not being compiled? I am testing it in an Angular development project.

Regards.

`[ng] ./node_modules/object-sizeof/indexv2.js:68:21-36 - Error: Module not found: Error: Can't resolve 'util' in 'C:\Users\Marius\Documents\Proyectos\memoshare-front\node_modules\object-sizeof'
[ng]
[ng] BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
[ng] This is no longer the case. Verify if you need this module and configure a polyfill for it.
[ng]
[ng] If you want to include a polyfill, you need to:
[ng]    - add a fallback 'resolve.fallback: { "util": require.resolve("util/") }'
[ng]    - install 'util'
[ng] If you don't want to include a polyfill, you can use an empty module like this:
[ng]    resolve.fallback: { "util": false }
[ng]
[ng]
[ng] × Failed to compile.
`

--- a/.travis.yml
+++ b/.travis.yml
@@ -1,7 +1,0 @@
-language: node_js
-node_js:
-  - v10
-  - v12
-  - v13
-  - v14
-  - v16

--- a/README.md
+++ b/README.md
@@ -1,6 +1,6 @@
 ## object-sizeof
 
-[![Build](https://img.shields.io/npm/v/object-sizeof)](https://img.shields.io/npm/v/object-sizeof) [![Build Status](https://travis-ci.org/miktam/sizeof.svg?branch=master)](https://travis-ci.org/miktam/sizeof) ![GitHub contributors](https://img.shields.io/github/contributors/miktam/sizeof) [![NPM](https://img.shields.io/npm/dy/object-sizeof)](https://img.shields.io/npm/dy/object-sizeof) [![codecov](https://codecov.io/gh/miktam/sizeof/branch/master/graph/badge.svg?token=qPHxmWpC1K)](https://codecov.io/gh/miktam/sizeof)
+[![Build](https://img.shields.io/npm/v/object-sizeof)](https://img.shields.io/npm/v/object-sizeof) ![GitHub contributors](https://img.shields.io/github/contributors/miktam/sizeof) [![NPM](https://img.shields.io/npm/dy/object-sizeof)](https://img.shields.io/npm/dy/object-sizeof) [![codecov](https://codecov.io/gh/miktam/sizeof/branch/master/graph/badge.svg?token=qPHxmWpC1K)](https://codecov.io/gh/miktam/sizeof)
 
 ### Get size of a JavaScript object in Bytes
 
@@ -16,7 +16,7 @@ Please note that this function will only work in some cases, especially when dea
 - Set
 - BigInt
 - Function
-  
+
 ### Coding standards
 
 Project follows [JavaScript Standard Style](https://standardjs.com/) as a JavaScript style guide.

--- a/indexv2.js
+++ b/indexv2.js
@@ -3,15 +3,6 @@
 'use strict'
 const ECMA_SIZES = require('./byte_size')
 
-/** Is Node.js environment */
-function isNode () {
-  return (
-    typeof process === 'object' &&
-    typeof process.versions === 'object' &&
-    typeof process.versions.node !== 'undefined'
-  )
-}
-
 /**
  * Size in bytes for complex objects
  * @param {*} obj
@@ -69,12 +60,7 @@ function objectSizeSimple (obj) {
     } else if (typeof value === 'bigint') {
       bytes += Buffer.from(value.toString()).byteLength
     } else if (typeof value === 'function') {
-      if (isNode()) {
-        const util = require('util')
-        bytes += Buffer.byteLength(util.inspect(value), 'utf8')
-      } else {
-        bytes += value.toString().length
-      }
+      bytes += value.toString().length
     } else if (typeof value === 'object' && objectList.indexOf(value) === -1) {
       objectList.push(value)
 

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
   "name": "object-sizeof",
-  "version": "2.4.1",
+  "version": "2.4.2",
   "description": "Sizeof of a JavaScript object in Bytes",
   "main": "indexv2.js",
   "scripts": {

--- a/test/test.js
+++ b/test/test.js
@@ -92,29 +92,11 @@ describe('sizeof node.js tests', () => {
     sizeof(BigInt(21474836480)).should.equal(11)
   })
 
-  it('Function support', () => {
-    const func = (one, two) => {
-      return one + two
-    }
-    sizeof(func).should.equal(16)
-  })
-
   it('nested objects', () => {
     const obj = { a: 1, b: 2, c: 3 }
     sizeof(obj).should.be.equal(19)
     const nested = { d: obj }
     sizeof(nested).should.be.equal(25)
-  })
-})
-
-describe('sizeof browser tests', function () {
-  const versions = global.process.versions
-  before(function () {
-    delete global.process.versions
-  })
-
-  after(function () {
-    global.process.versions = versions
   })
 
   it('Function support', () => {

